src/process-columnar-results.js

```diff
@@ -118,9 +119,16 @@ export default function processColumnarResults(
           case "TIME":
           case "TIMESTAMP":
           case "DATE":
-            row[fieldName] = new Date(
-              data.columns[c].data.int_col[r] * oneThousandMilliseconds
-            )
+            // The JS date constructor expects time as 'number of ms since the epoch'. The raw
+            // integer value in the DB may represent s, ms, us, or ns. We read the precision of
+            // the column to figure out what the DB value represents, then convert that to ms.
+
+            // A precision of 0 = sec, 3 = ms. Thus, this line finds the value to divide the DB val
+            // eslint-disable-next-line no-magic-numbers
```

Comment:
I don't like single-line disables like this. However, I explain the origin of the "magic" values in the comment immediately preceding this, and I feel like assigning these values to a variable first before using them would only make the line more confusing. So, in this instance, I feel disabling the warning adheres to the spirit of the rule.